aten/src/ATen/gen_backend_select_register.py

```diff
@@ -0,0 +1,64 @@
+# This script generates BackendSelectRegister.cpp which is being used for dispatching purposes.
+# We process only those factory functions that have 'backend_select' flag in its native_functions.yaml definition.
+
+from code_template import CodeTemplate
+
+GENERATED_COMMENT = CodeTemplate(
+    "@" + "generated from ${filename}")
```

Comment:
the other examples of this do this via an entry in the template, right?

Reply:
Yeah, its being [set once](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/gen.py#L118) and used everywhere.

Reply:
I meant in the template file, not in the codegen.  I don't see any reason for this to be in the codegen.  See e.g: https://github.com/pytorch/pytorch/blob/8195961f20caa18863698d85410d0818bfd67463/aten/src/ATen/templates/Functions.h#L3

Reply:
ah, I guess you aren't passing around the env (from the top-layer) so you can't use it.